frontend/src/components/editor/ManifestUploader.js

```diff
@@ -64,27 +65,33 @@ class ManifestUploader extends React.Component {
     }
   };
 
-  compareSections = (operator, merged) => {
+  compareSections = (operator, merged, uploaded) => {
     const { markSectionForReview } = this.props;
 
     Object.keys(sectionsFields).forEach(sectionName => {
       const fields = sectionsFields[sectionName];
-      let same = true;
+      let updated = false;
 
       // check if operator fields are same as before upload
       if (typeof fields === 'string') {
-        same = _.isEqual(_.get(operator, fields), _.get(merged, fields));
+        updated = this.operatorFieldWasUpdated(fields, operator, uploaded, merged);
       } else {
-        same = fields.every(path => _.isEqual(_.get(operator, path), _.get(merged, path)));
+        updated = fields.some(path => this.operatorFieldWasUpdated(path, operator, uploaded, merged));
       }
 
-      if (!same) {
-        // if not mark section as needing review
+      if (updated) {
+        // mark section as review needed
         markSectionForReview(sectionName);
       }
     });
   };
 
+  operatorFieldWasUpdated = (fieldName, operator, uploadedOperator, mergedOperator) =>
+    // field changed when either its value changed
+    // or uploaded operator was same as default values
+    !_.isEqual(_.get(operator, fieldName), _.get(mergedOperator, fieldName)) ||
+    _.isEqual(_.get(defaultOperator, fieldName), _.get(uploadedOperator, fieldName));
```

Comment:
I think we should check for a value here. If both the default and the uploaded values are blank, then I would assume there is no update here.

Reply:
yep, good point here